vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -949,6 +987,20 @@ AdsImpl::GetPageScoreCache() const {
   return page_score_cache_;
 }
 
+std::vector<std::string> AdsImpl::GetWinningPurchaseIntentCategories() {
```

Comment:
Could we add `using WinningPurchaseIntentCategories = std::vector<std::string>;` and then through-out the codebase use this instead of `std::vector<std::string>`, which in turn will remove the need for `auto`?